packages/client/components/ActivityLibrary/ActivityDetails/ActivityDetails.tsx

```diff
@@ -52,6 +52,7 @@ export const query = graphql`
       }
       teams {
         id
+        tier
```

Comment:
+1 why do we need to query `isFree` here?

Reply:
🤔 this must've been leftover from an earlier version - removed!